examples/cosmos/sdk_boundary_test.go

```diff
@@ -0,0 +1,132 @@
+package cosmos_test
+
+import (
+	"context"
+	"testing"
+
+	interchaintest "github.com/strangelove-ventures/interchaintest/v8"
+	"github.com/strangelove-ventures/interchaintest/v8/chain/cosmos"
+	"github.com/strangelove-ventures/interchaintest/v8/conformance"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/strangelove-ventures/interchaintest/v8/relayer"
+	"github.com/strangelove-ventures/interchaintest/v8/relayer/rly"
+	"github.com/strangelove-ventures/interchaintest/v8/testreporter"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+type boundarySpecs struct {
+	name           string
+	chainSpecs     []*interchaintest.ChainSpec
+	relayerVersion string
+}
+
+func TestSDKBoundaries(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping in short mode")
+	}
+
+	t.Parallel()
+
+	var tests = []boundarySpecs{
+		{
+			name: "sdk 45 <-> 50",
+			chainSpecs: []*interchaintest.ChainSpec{
+				{
+					Name: "gaia", ChainName: "gaia", Version: "v7.0.3", //sdk 0.45.6
+				},
+				{
+					Name: "ibc-go-simd", ChainName: "simd-50", Version: "feat-upgrade-sdk-v0.50", //sdk 0.50 alpha
+					ChainConfig: ibc.ChainConfig{
+						UsingNewGenesisCommand: true,
+					},
+				},
+			},
+			relayerVersion: "colin-event-fix",
```

Comment:
qq: do you guys have an eta for when the golang relayer going to be updated with the changes in @colin-axner's image?

Reply:
just approved colin's pr! should be merged once CI passes